File: lib_glide/src/main/java/com/third/image/glide/GlideUtil.java
package com.third.image.glide;

import android.net.Uri;
import android.util.Log;
import android.widget.ImageView;

import com.bumptech.glide.Glide;
import com.lib.annotation.Insert;

public class GlideUtil {
    private static final String TAG = GlideUtil.class.getSimpleName();

    @Insert(classPath = "com.drcuiyutao.lib.image.ImageUtil", name = "displayImage", replace = true)
    public static void displayImage(String url, ImageView view) {
        Log.i(TAG, "displayImage use Glide!!!");
        Glide.with(view.getContext()).load(Uri.parse(url)).override(306, 600).fitCenter().into(view);
    }
}

File: lib_glide/src/main/java/com/third/image/glide/GlideUtil.kt
package com.third.image.glide

import android.net.Uri
import android.util.Log
import android.widget.ImageView

import com.bumptech.glide.Glide
import com.lib.annotation.Inject

object GlideUtil {
//    private val TAG = GlideUtil::class.java.simpleName

    @Inject(classPath = "com.study.lib.image.ImageUtil", name = "displayImage", replace = true)
    fun displayImage(url: String, view: ImageView) {
        Log.i("GlideUtil", "displayImage use Glide!!!")
        Glide.with(view.context)
            .asBitmap()
            .load(Uri.parse(url))
            .override(640, 360)
            .fitCenter()
            .into(view)
    }
}
